README.md

```diff
@@ -40,15 +52,91 @@ or
 python setup.py develop
 ```
 
+## Getting started
+`s2spy` provides end-to-end solutions for machine learning (ML) based S2S forecasting.
+
+![workflow](./docs/assets/images/workflow.png)
+
+### Datetime operations & Data processing
+In a typical ML-based S2S project, the first step is always data processing.  A calendar-based datetime operation module is implemented to help the users prepare their data. For instance, if a user is looking for predictors for winter climate at seasonal timescales (~180 days), a `AdventCalendar` can be used to prepare the data:
+
+```py
+calendar = s2spy.time.AdventCalendar(anchor=(11, 30), freq='180d')
+calendar = calendar.map_years(2020, 2021)
+calendar
```

Comment:
```suggestion
calendar.show()
```

Reply:
I think here `calendar.show()` should be used because the output of `calendar` is `AdventCalendar(month=11, day=30, freq=180d, n_targets=1)`